linux_os/guide/services/xwindows/disabling_xwindows/package_xorg-x11-server-common_removed/rule.yml

```diff
@@ -9,8 +9,10 @@ description: |-
     installed. If X Windows is not installed then the system cannot boot into graphical user mode.
     This prevents the system from being accidentally or maliciously booted into a <tt>graphical.target</tt>
     mode. To do so, run the following command:
-    <pre>$ sudo yum groupremove "X Window System"</pre>
-    <pre>$ sudo yum remove xorg-x11-server-common</pre>
+    {{%- if product != "rhel8" and product != "rhv4" -%}}
+    <pre>$ sudo {{{ pkg_manager }}} groupremove "X Window System"</pre>
```

Comment:
"X Window System" group is not available on RHEL8

Reply:
What is the RHEL 8 equivalent?

Reply:
It seems to be `base-x` they have similar package list, from RHEL8 machine:

```
[root@localhost gen]# yum group info base-x
Updating Subscription Management repositories.
Unable to read consumer identity
This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.
Last metadata expiration check: 0:04:26 ago on Wed 22 Apr 2020 11:55:27 AM EDT.

Group: base-x
 Description: Local X.org display server
 Mandatory Packages:
   glx-utils
   mesa-dri-drivers
   plymouth-system-theme
   xorg-x11-drv-ati
   xorg-x11-drv-evdev
   xorg-x11-drv-fbdev
   xorg-x11-drv-intel
   xorg-x11-drv-libinput
   xorg-x11-drv-nouveau
   xorg-x11-drv-qxl
   xorg-x11-drv-vesa
   xorg-x11-drv-vmware
   xorg-x11-drv-wacom
   xorg-x11-server-Xorg
   xorg-x11-utils
   xorg-x11-xauth
   xorg-x11-xinit
   xorg-x11-xinit-session
```

From RHEL7 machine:
```
[root@localhost ~]# yum group info 'X Window System'
Loaded plugins: product-id, search-disabled-repos, subscription-manager
This system is not registered with an entitlement server. You can use subscription-manager to register.
There is no installed groups file.
Maybe run: yum groups mark convert (see man yum)

Group: X Window System
 Group-Id: x11
 Description: X Window System Support.
 Mandatory Packages:
    glx-utils
   +initial-setup-gui
    mesa-dri-drivers
   +plymouth-system-theme
   +spice-vdagent
   +xorg-x11-drivers
   +xorg-x11-server-Xorg
   +xorg-x11-utils
    xorg-x11-xauth
    xorg-x11-xinit
   +xvattr
 Optional Packages:
   mesa-libGLES
   tigervnc-server
   wayland-protocols-devel
   xorg-x11-drv-evdev
   xorg-x11-drv-keyboard
   xorg-x11-drv-libinput
   xorg-x11-drv-mouse
   xorg-x11-drv-openchrome
```

Funny thing is that neither of those groups include `xorg-x11-server-common` package.
Removing this package manually remove many other dependencies: https://pastebin.com/yd1LwvyZ

Reply:
I selected the package group `base-x` for RHEL8 based products.